src/main/java/jnr/ffi/provider/jffi/AsmLibraryLoader.java

```diff
@@ -20,6 +20,18 @@
 package jnr.ffi.provider.jffi;
 
 import com.kenai.jffi.Function;
+
+import org.objectweb.asm.ClassReader;
+import org.objectweb.asm.ClassVisitor;
+import org.objectweb.asm.ClassWriter;
+
+import java.io.PrintWriter;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.lang.reflect.ParameterizedType;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
+
 import jnr.ffi.CallingConvention;
```

Comment:
These imports appear to have been out of order anyway, but in the future try to do non-code changes (whitespace, import cleanup, etc) as separate commits.